sql/core/src/test/scala/org/apache/spark/sql/execution/BenchmarkWholeStageCodegen.scala

```diff
@@ -148,7 +197,8 @@ class BenchmarkWholeStageCodegen extends SparkFunSuite {
   }
 
   test("benchmark") {
```

Comment:
Shall we add a comment and say those benchmarks are skipped in normal build, or change `test("benchmark")` to `ignore("benchmark")` to be clear.